### ISSUE TITLE
Multiple tab bar button problem..

### ISSUE THREAD

**Issue by ismaiI1** [Yalantis/ColorMatchTabs#75]
# Report

###  What do you want to add?

> I want to add dynamic UIBarButton and want to change title by some conditions. I can add button but when i called "reloadData()" it's adding tab bar buttons again.. You can see in screen record gif..

![ezgif-1-a7405970e9b8](https://user-images.githubusercontent.com/8086679/48617799-7c0ffc00-e9a8-11e8-9707-e16eaa1fb1ab.gif)

### Your Environment

- Version of the component: 3.1
- Swift version: 4.0
- iOS version: 12.1
- Device: Iphone 8plus
- Xcode version: 10
- 
### Stack

```
   CocoaPods : 1.5.3
        Ruby : ruby 2.3.7p456 (2018-03-28 revision 63024) [universal.x86_64-darwin17]
    RubyGems : 2.5.2.3
        Host : Mac OS X 10.13.6 (17G3025)
       Xcode : 10.1 (10B61)
         Git : git version 2.17.2 (Apple Git-113)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ 273dd3e282276ad3c1138ccbebf4f374af910a32
               twilio - https://github.com/twilio/cocoapod-specs @ 15612163b2cae5be50339701ce3702af12a090f8
```

### Installation Source

```
Executable Path: /usr/local/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.2
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.3.0
cocoapods-try         : 1.1.0
```

**Comment by ismaiI1** [comment]
@rnkyr Menu labels was removing when call `reloadData()`

**Comment by den-zv** [comment]
Closed via #78
